### PR TITLE
Feature/bphh 394/reviewer inbox

### PR DESCRIPTION
**Pull request by bkmaibach**
Reset DB for new seeds
Log in as reviewer, go to submission inbox
behold the table and rejoice

What type of PR is this? (check all applicable)
[ x] 🍕 Feature
 🐛 Bug Fix
 📦 Chore (Release)
 ✅ Test

Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-394
https://hous-bssb.atlassian.net/browse/BPHH-533

### CODE REVIEW

File: app/controllers/api/permit_applications_controller.rb [hunk 1]
```diff
@@ -1,10 +1,7 @@
 class Api::PermitApplicationsController < Api::ApplicationController
+  include Api::Concerns::Search::JurisdictionPermitApplications
   before_action :set_permit_application, only: %i[show update] #destroy
-
-  def index
-    @permit_applications = policy_scope(PermitApplication)
-    render_success @permit_applications, nil, { blueprint: PermitApplicationBlueprint }
-  end
+  before_action :set_jurisdiction, only: %i[index]
```
Comment (taw27) [inline app/controllers/api/permit_applications_controller.rb:4]: is this correct, seems like there is no index action

File: app/models/permit_application.rb [hunk 2]
```diff
@@ -29,8 +46,48 @@ def requirements
...
+        )
+    else
+      # Start with the initial number if there are no previous numbers
+      new_number = format("%s-001-000-000", number_prefix)
+    end
+
+    # Assign the new number to the permit application
+    self.number = new_number
+    return new_number
+  end
```
Comment (taw27) [inline app/models/permit_application.rb:82]: What is the goal of this? Can you give some examples

**Review by taw27 — CHANGES_REQUESTED** [review]

File: app/models/permit_application.rb (hunk 2 above)
Comment (bkmaibach) [reply app/models/permit_application.rb:82]: Sure!:

The objective is to create a unique number for each application.

like CN-001-111-111
then CN-001-111-112

I may need to test this more but initial results are good

Comment (bkmaibach) [reply app/models/permit_application.rb:82]: im going to fix up the clarity of this a bit

Comment (bkmaibach) [reply app/models/permit_application.rb]: image

**Review by taw27 — APPROVED** [review]

**Pull request merged**